tests/range/range_api.cpp

```diff
@@ -307,21 +226,25 @@ class TEST_NAME : public util::test_base {
    */
   void run(util::logger &log) override {
     {
+      log.note(
+          "ComputeCpp does not implement unary minus operation. "
+          "Skipping this test case.");
```

Comment:
I think this code should be wrapped in `SYCL_CTS_COMPILING_WITH_COMPUTECPP` macro and use Catch2 macro `SKIP` or `WARN` instead of `log`:
```c++
#ifdef SYCL_CTS_COMPILING_WITH_COMPUTECPP
  WARN(...)
#endif
```

Reply:
I used the old framework here to keep it consistent with the existing file, but I see that there is no reason to do so. I have replaced it as you suggest, but I have not used the `SKIP` macro as that would skip the entire test. The warning is now:
```c++
      WARN(
          "ComputeCpp does not implement unary minus operation. "
          "Skipping the test for this operation.");
```